services-js/commissions-search/src/FacetList.js

```diff
@@ -14,47 +17,63 @@ const GET_AREAS = gql`
   }
 `;
 
-class FacetList extends Component {
-  renderArea() {
-    function Area(props) {
-      return (
-        <label className="cb">
-          <input
-            id={`checkbox-area-${props.id}`}
-            name={props.id}
-            type="checkbox"
-            value={props.id}
-            className="cb-f"
-            checked={props.checked}
-            onChange={props.handleCheckChange}
-          />
-          <span className="cb-l">{props.name}</span>
-        </label>
-      );
-    }
+function returnArea(props, policy_area) {
+  // to conserve horizontal space, convert "and" to an ampersand; Oxford comma
+  // looks odd in this case, and the Chicago Manual of Style agrees - jm
+  const labelText = policy_area.name.replace(/(and|, and)/, '&');
```

Comment:
Doesn’t this remove the space in the Oxford comma situation? "A, and B" -> "A& B"

Reply:
oops - you're so right.